force-app/main/default/classes/DMLOperations.cls

```diff
@@ -145,11 +174,45 @@ public with sharing class DMLOperations {
      * @param oppNames The list of Opportunity names.
      */
     public static void upsertOpportunities(String accountName, List<String> oppNames) {
+        // Try to find the account
+        Account account;
+        try {
+            account = [SELECT Id FROM Account WHERE Name = :accountName LIMIT 1];
+        } catch (QueryException e) {
+            account = null;
+        }
```

Comment:
I wouldnt recommend using try catch here because there are other query exceptions that might happen that might not be exactly what you are looking for. You just want to know if there are no results. Is there another way we reviewed to check if a query returns no results?